client/CVideoHandler.cpp

```diff
@@ -672,8 +679,24 @@ bool CVideoPlayer::playVideo(int x, int y, bool stopOnKey)
 	return true;
 }
 
-bool CVideoPlayer::openAndPlayVideo(const VideoPath & name, int x, int y, bool stopOnKey, bool scale)
+bool CVideoPlayer::openAndPlayVideo(const VideoPath & name, int x, int y, EVideoType videoType)
 {
+	bool scale;
+	bool stopOnKey;
+
+	switch(videoType)
+	{
+		case EVideoType::INTRO:
+			stopOnKey = true;
+			scale = true;
+			overlay = false;
+			break;
+		case EVideoType::SPELLBOOK:
+		default:
+			stopOnKey = false;
+			scale = false;
+			overlay = true;
+	}
 	open(name, false, true, scale);
 	bool ret = playVideo(x, y,  stopOnKey);
```

Comment:
Should overlay be a `CVideoPlayer::playVideo()` argument?

Reply:
Either way would work I think?
Function parameter might be more logical